README.md

```diff
@@ -1 +1,60 @@
-# URL Shrtn
\ No newline at end of file
+# Shrtn Ur URL
+
+A simple URL shortener built with Python/Flask, perfect for all the lazy web surfers out there! Users can submit any https:// formatted URLs and obtain a shortened version of it. For example the URL for the Flask User Guide, https://flask.palletsprojects.com/en/1.1.x/#user-s-guide has been shortened to https://shrtn-ur-url.herokuapp.com/hyXDp
+
+This app was created by [@roselynle](https://github.com/roselynle) and [@FaisalY12](https://github.com/FaisalY12) as part of Futureproof's Coding Challenge.
+
+### Installation
+
+-   Clone or download this repo 
+
+### Usage
+
+-   Open your terminal and navigate to the `url-shrtn` folder
+-   Run `pipenv shell`
+-   Run `pipenv install`
+-   Run `pipenv run dev`
+-   Flask API will be running on port 5000
+
```

Comment:
good instructions - I needed to install colorama module too in my shell but might be an issue with my environment.